src/utils/metrics.ts

```diff
@@ -66,8 +66,10 @@ export const deleteMetric = async (
   scorerName: string,
   scorerType: ScorerTypes
 ): Promise<void> => {
-  const scorers = await getScorers(scorerType);
-  const scorer = scorers.find((s) => s.name === scorerName);
+  const names: string[] = [scorerName];
+  console.log('Deleting metric with names:', names);
+  const scorers = await getScorers(scorerType, names);
+  const scorer = scorers[0]; // There should only ever be one here
```

Comment:
will this cause an error?  should the check below be updated to `if(scorers.length === 0)` and moved above this line?